src/blog/_posts/2018-11-06-opera-57.md

```diff
@@ -0,0 +1,151 @@
+---
+title: What's new in Chromium 70 and Opera 57
+authors:
+- fredrik-soderqvist
+intro: 'Opera 57 (based on Chromium 70) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To find out what’s new for users, see our [Desktop blog post](https://blogs.opera.com/desktop/2018/11/opera-57-stable). Here’s what it means for web developers.'
+tags:
+- html
+- css
+- dom
+- javascript
+- opera
+license: cc-by-3.0
+---
+
+Opera 57 (based on Chromium 70) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To
+find out what’s new for users, see our
+[Desktop blog post](https://blogs.opera.com/desktop/2018/11/opera-57-stable).
+Here’s what it means for web developers.
+
+## Displaying a dialog will cause pages to leave fullscreen
+
+Dialogs (authentication prompt, payment request et.c) require proper context
+when shown to the user. When the browser is in fullscreen mode - an immersive
+mode - that context is not available. To help the user get the right context
+when a dialog is displayed, the browser will now leave fullscreen mode in those
+cases.
+
+## The `ontouch*` APIs default to disabled on desktop platforms
+
+The `ontouch*` event attributes on `Window`, `Document` and `Element` are
+now disabled by default on desktop platforms. This is to avoid confusion when
+feature detecting touch support - use `navigator.maxTouchPoints` for
+touchscreen detection and `window.TouchEvent` touch event detection instead. To
+support both touch and mouse input, consider using the [Pointer Events](https://w3c.github.io/pointerevents/) API.
+
+Event listeners for the associated touch events can still be added via
+`EventTarget.prototype.addEventListener`, like:
+
+    document.addEventListener("touchstart", ...);
+
+## Options dictionary for postMessage methods
```

Comment:
To be consistent postMessage should be highlighted, or Symbol.prototype.description in a header below should not be.

Reply:
Highlighted (not sure that makes a difference though - style-wise.)